compiler/src/dotty/tools/dotc/quoted/QuoteDriver.scala

```diff
@@ -11,12 +11,13 @@ import scala.quoted.Expr
 
 import java.net.URLClassLoader
 
+import Runners.{Settings, Run, Show}
+
 class QuoteDriver extends Driver {
   import tpd._
 
-  def run[T](expr: Expr[T], settings: Runners.RunSettings): T = {
-    val ctx: Context = initCtx.fresh
-    ctx.settings.optimise.update(settings.optimise)(ctx)
+  def run[T](expr: Expr[T], settings: Settings[Run]): T = {
+    val (_, ctx: Context) = setup(settings.compilerArgs.toArray :+ "dummy.scala", initCtx.fresh)
```

Comment:
`dummy.scala`?

Reply:
`setup` expects some input file. But it is not required here. Hence I add a dummy file name that is then ignored. It is the `_` in `val (_, ctx: Context) = `